.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+Github Issue: #NUM _(replace NUM with Github issue number)_
+
+### Description
+
+_Summarize the ticket here_
+
+### Overview of work done
+
+_Summarize the work you did_
+
+### Overview of verification done
+
+_Summarize the testing and verification you've done. This includes unit tests or testing with specific data_
+
+### Overview of integration done
+
+_Explain how this change was integration tested. Provide screenshots or logs if appropriate. An example of this would be a local Harmony deployment._
+
+## PR checklist:
+
+* [ ] Linted
+* [ ] Updated unit tests
+* [ ] Updated changelog
+* [ ] Integration testing
+
```

Comment:
Do we need to add a step for created pull request for updated documentation (if applicable) ?

Reply:
Doesn't look like this repo has docs yet. If we set that up, then I would add it to the checklist in the template

Reply:
Our documentation is currently in another repo. It is a more general documentation for the maap platform, but we should update those and any tutorials if making a significant change that could impact how users use this module. You're right we don't have one for this repo.

Reply:
@sujen1412 added an item for documentation